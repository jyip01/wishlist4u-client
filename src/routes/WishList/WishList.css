.Button__AddWish{
    background-color: rgba(228, 228, 250, 0.831);
    border: 1px solid rgba(31, 31, 41, 0.824);
    border-radius: 6px;
    color: rgb(37, 35, 58);
    font-size: 13px;
    text-shadow: rgba(255, 255, 255, 0.995);
    margin-bottom: 5px;
    font-weight: bold;
  }

.WishList h2{
  text-shadow: 0 0 3px #F00000;
  font-weight: bold;
  text-decoration: underline;
  color:white;
}

.WishList p{
  color: black;
  font-size: 18px;
}
