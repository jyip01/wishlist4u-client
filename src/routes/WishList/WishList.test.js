import React from 'react';
import ReactDOM from 'react-dom';
import WishList from './WishList';
import {BrowserRouter} from 'react-router-dom';
import renderer from 'react-test-renderer';

it('renders without crashing',()=>{
    const div = document.createElement('div')
    ReactDOM.render(<BrowserRouter><WishList/></BrowserRouter>,div)
    ReactDOM.unmountComponentAtNode(div)
})

it('should match with snapshot', () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <WishList/>
        </BrowserRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot()
})
