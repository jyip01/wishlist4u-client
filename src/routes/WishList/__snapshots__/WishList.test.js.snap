// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match with snapshot 1`] = `
<section
  className="Section WishList card"
>
  <div
    className="loading"
  />
</section>
`;
